### PR TITLE
Created the Shivam-officials.json file

### PR DESCRIPTION
**Pull request by Shivam-officials**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1864"></a>

### CODE REVIEW

File: public/data/Shivam-officials.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Shivam",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/Shivam-officials.png",
+  "links": [
+    {
+      "name": "Waiting for u on GitHub",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
```
Comment (Panquesito7) [inline public/data/Shivam-officials.json:9]:
```suggestion
      "url": "https://github.com/Shivam-officials",
```

File: public/data/Shivam-officials.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Starting Freelancing",
+      "date": "2 October 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (Panquesito7) [inline public/data/Shivam-officials.json:25]: You might want to remove or update this link here.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Shivam-officials — COMMENTED** [review]
i updated the url plz add review it once more

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**